changelog/unreleased/changes/2087--msgpack-deprecation.md

```diff
@@ -0,0 +1,3 @@
+The `msgpack` encoding option is now deprecated. VAST issues a warning on
+startup and automatically uses the `arrow` encoding instead. A future version of
+VAST will remove this option entirely.
```

Comment:
Still needs a separate changelog entry for aging deprecation and the separate PR against our docs.